WordPressKit.podspec

```
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressKit'
-  s.version       = '8.4.0'
+  s.version       = '8.5.0-beta.1'
```

This will need to be bumped due to #611 using the same version:
```suggestion
  s.version       = '8.5.0-beta.2'
```